governance/program/src/state/governance.rs

```diff
@@ -309,8 +313,17 @@ pub fn get_governance_data(
         // For legacy accounts set the community Veto threshold to Disabled
         governance_data.config.community_veto_vote_threshold = VoteThreshold::Disabled;
 
-        // Reset reserved space previously used for voting_proposal_count
-        governance_data.config.reserved = [0; 5];
+        // Reset voting_cool_off_time and reserved space previously used for voting_proposal_count
+
+        // Default voting_cool_off_time to 1h for max_voting_time >= 10h
+        governance_data.config.voting_cool_off_time =
+            if governance_data.config.max_voting_time >= 36000 {
+                3600
+            } else {
+                0
+            };
```

Comment:
Sorry, I still don't understand the reason for making a default change to all existing governances.  What's the problem with an avalanche of governances trying to make this change? I would be more worried about my proposal's voting time changing all of a sudden

Reply:
mainly operational, a lot of DAO's would require support from labs in order to upgrade their realm

Reply:
My thinking was that it was a recommended setting and really all DAOs should use it to increase their security and without the default override most of them would never make the change. 

However given the potential confusion I backed out and removed the default and would increase awareness among the DAOs instead.